docs/index.rst

```diff
@@ -1,12 +1,38 @@
-************
-Introduction
-************
-AMD **ROCm** has two classification of libraries,
+.. meta::
+  :description: hipBLAS documentation and API reference library
+  :keywords: hipBLAS, rocBLAS, BLAS, ROCm, API, Linear Algebra, documentation
 
-- **roc**\* : AMD GPU Libraries, written in `HIP <https://docs.amd.com/bundle/AMD_HIP_Programming_Guide/page/Introduction.html>`_.
-- **hip**\* : AMD CPU library that is a thin interface to either AMD **roc*** or Nvidia **cu*** libraries.
+.. _hipblas:
 
-Users targeting both CUDA and AMD devices must use the **hip*** libraries.
+********************************************************************
+hipBLAS documentation
+********************************************************************
 
 hipBLAS is a BLAS marshaling library with multiple supported backends. It sits between the application and a 'worker' BLAS library, marshalling inputs into the backend library and marshalling results back to the application.
 hipBLAS exports an interface that does not require the client to change, regardless of the chosen backend. Currently, it supports `rocBLAS <https://github.com/ROCmSoftwarePlatform/rocBLAS>`_ and `cuBLAS <https://developer.nvidia.com/cublas>`_ as backends.
+
+The code is open and hosted at: https://github.com/ROCm/hipBLAS
+
+The hipBLAS documentation is structured as follows:
+
+.. grid:: 2
+
+  .. grid-item-card:: Installation
+
+    * :ref:`install`
+
+  .. grid-item-card:: How-To
+
+    * :ref:`hipblas-orga`
+    * :ref:`hipblas-clients`
+    * :ref:`contribute`
+
+  .. grid-item-card:: Reference
+
+    * :ref:`api_label`
+    * :ref:`deprecations`
+
```

Comment:
This doesn't display as intended in the "rich diff" in github. Might just be a github limitation, but make sure it displays correctly in the build for you.

Reply:
It looks correct in the generated HTML page.